lib/api/core/pluginsContext.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  ResponseObject: require('./models/responseObject'),
+  RealTimeResponseObject: require('./models/realTimeResponseObject'),
+
+  KuzzleError: require('./errors/kuzzleError'),
+  BadRequestError: require('./errors/badRequestError'),
+  ForbiddenError: require('./errors/forbiddenError'),
+  InternalError: require('./errors/internalError'),
+  NotFoundError: require('./errors/notFoundError'),
+  PartialError: require('./errors/partialError'),
+  ServiceUnavailableError: require('./errors/serviceUnavailableError'),
+  UnauthorizedError: require('./errors/unauthorizedError')
```

Comment:
yes, good idea.

so we could have someting like :
module.exports = {
  ResponseObject: require('./models/responseObject'),
  RealTimeResponseObject: require('./models/realTimeResponseObject'),
  errors: require('./errors')
};

and then, in the plugin controller, create an error  with :
var error = new context.errors.InternalError();

is it what you mean ?